packages/playground-server/src/node.ts

```diff
@@ -124,6 +102,7 @@ export default class NodeWrapper {
       {
         STORE_KEY_PREFIX: "store"
       },
+      // @ts-ignore
```

Comment:
Any better alternative than ignoring?

Reply:
Yeah, removing that, lol. It's not even necessary anymore.